salt/states/grains.py

```diff
@@ -169,26 +169,29 @@ def list_absent(name, value):
               - dev
     '''
     ret = {'name': name,
-           'changes': {},
+           'changes': [],
```

Comment:
@bstevenson, thanks for working on this.  `changes` needs to be a dictionary and `comment` needs to be a string as described in the [state docs](https://docs.saltstack.com/en/latest/ref/states/writing.html#return-data).